### ISSUE TITLE
Failed to create native "BaseComm_IsClientGagged", name is probably already in use

### ISSUE THREAD

**Issue by valeologist** [SB-MaterialAdmin/NewServer#86]
список плагинов:
  ```
01 "Client Preferences" (1.10.0.6460) by AlliedModders LLC
  02 "[Levels Ranks] Core" (v3.1.1) by RoadSide Romeo & Wend4r
  03 "Knife ghost error fix" (1.1) by Phoenix & Wend4r
  04 "Action if say.." (1.0.0) by OkyHp
  05 "[Scoreboard] Language" (1.3) by Wend4r
  06 "Reweapon" (1.1.0) by Phoenix, Tonki_Ton
  07 "Anti-Flood" (1.10.0.6460) by AlliedModders LLC
  08 "No Blood" (1.0.1) by Bara
  09 "Ограничение оружия" (1.2.0) by Phoenix (˙·٠●Феникс●٠·˙)
  10 "Basic Comm Control" (0.7.9) by AlliedModders LLC
  11 "gravity.smx"
  12 "Sound Commands" (1.10.0.6460) by AlliedModders LLC
  13 "Реклама" (1.4) by Pheonix (˙·٠●Феникс●٠·˙)
  14 "Basic Chat" (1.10.0.6460) by AlliedModders LLC
  15 "Material Admin Checker" (0.7.9) by psychonic, Ca$h Munny, Sarabveer(VEER™)
  16 "Weapon Skin" (2.3.10 PRIVATE) by Phoenix (˙·٠●Феникс●٠·˙)
  17 "Map configs" (1.3) by Berni
  18 "[LR] Module - Calibration" (v3.1) by RoadSide Romeo
  19 "[LR WEB] Chat" (1.0.0) by OkyHp, SAPSAN
  20 "[LR] Module - Unusual Kills" (v3.1) by Wend4r
  21 "[LR] Module - FakeRank" (v3.1) by RoadSide Romeo & Wend4r
  22 "[LR] Module - ExStats Hits" (v3.1) by Wend4r
  23 "[LR] Module - ExStats Maps" (v3.1) by RoadSide Romeo
  24 "[LR] Module - Top Vip Score" (v3.1) by Designed (Discord: .Designed#7985)
  25 "[LR] Module - Overlays" (v3.1) by RoadSide Romeo
  26  "[LR] Module - ExStats GeoIP" (v3.1) by RoadSide Romeo
  27 "[LR] Module - WEB" (v2.5.4.2) by Wend4r
  28 "[LR] Module - ExStats Weapons" (v3.1) by RoadSide Romeo & Wend4r
  29 "Gloves" (1.6.1) by Phoenix (˙·٠●Феникс●٠·˙)
  30 "[CS:GO] ResetScore" (1.0) by pracc
  31 "Quake Sounds v3" (3.5.0) by Spartan_C001
  32 "Player Commands" (1.10.0.6460) by AlliedModders LLC
  33 "Extended Player Death Info" (1.1.0) by Phoenix (˙·٠●Феникс●٠·˙), Icons created Bastaz
  34 "Reserved Slots" (1.10.0.6460) by AlliedModders LLC
  35 "Knife choice CORE - Выбор ножа ЯДРО" (2.11) by Phoenix - Феникс
  36 "Special Team Balancer" (0.6.0) by AlmazON
  37 "[Gifts] Core" (3.2) by R1KO
  38 "GameVoting" (1.1) by Neatek
  39 "Fun Commands" (1.10.0.6460) by AlliedModders LLC
  40 "Admin File Reader" (1.10.0.6460) by AlliedModders LLC
  41 "Fun Votes" (1.10.0.6460) by AlliedModders LLC
  42 "Basic Commands" (1.10.0.6460) by AlliedModders LLC
  43 "IP Name & Say Block" (2.0) by R1KO
  44 "bot_quota_0.smx"
  45 "Material Admin" (0.7.9) by Material Admin Dev Team
  46 "Admin Help" (1.10.0.6460) by AlliedModders LLC
  47 "Basic Votes" (1.10.0.6460) by AlliedModders LLC
  48 "Basic Info Triggers" (1.10.0.6460) by AlliedModders LLC
  49 "autodemo" (2.2.4) by mukunda
  50 "res/precacher.smx"
  51 "Steam Works Group Manager" (1.5) by Someone
  52 "VIP Hitmarker" (1.0.6) by Samolet, OkyHp
  53 "[VIP] Electro Effects" (1.0.3) by R1KO
  54 "[VIP] Stats" (2.6) by R1KO 
  55 "[VIP] GrenadeTrails" (1.0.1) by R1KO
  56 "[VIP] Vips Online" (1.0.1) by R1KO (skype: vova.andrienko1)
  57 "[VIP] Test" (1.0.3) by R1KO (skype: vova.andrienko1)
  58 "[VIP] CHAT (+scp)" (3.1.1) by R1KO
  59 "[VIP] Spawn Effects" (1.2) by Mr.ToNik
  60 "[VIP] MKIT" (1.1.3) by Pheonix (edit Felya)
  61 "[VIP] Skins" (1.0.5) by R1KO (skype: vova.andrienko1)
  62 "[VIP] Shield" (1.3.3 [PUBLIC]) by Faya™ (DS: Faya™#8514)
  63 "[VIP] Admin" (1.1) by R1KO
  64 "[VIP] Drop Weapons" (1.1) by R1KO
  65 "[VIP] Core" (3.0 R) by R1KO
  66 "Nextmap" (1.10.0.6460) by AlliedModders LLC
  67 "SteamWorks Additive Glider" (1.0) by Kyle Sanderson
  68 "Knife choice INTERFASE MONUL - Выбор ножа МОДУЛЬ ИНТЕРФЕЙСА" (1.1) by Phoenix - Феникс
  69 "Admin Menu" (0.7.9) by AlliedModders LLC
  70 "[Discord] Admin Log" (0.3.1) by Lord FEAR
  71 "[Discord] Admin Session Log" (1.0) by CrazyHackGUT aka Kruzya
  72 "[Discord] Core" (1.0.8) by CrazyHackGUT aka Kruzya
  73 "[Discord] SourceBans + SourceComms" (1.2.2) by CrazyHackGUT aka Kruzya
```
Версия sourcemod:
`sourcemod-1.10.0-git6460-linux.tar`

Лог ошибок
```
L 01/13/2020 - 13:29:22: SourceMod error session started
L 01/13/2020 - 13:29:22: Info (map "workshop/1177199224/de_dust2_2x2") (file "/home/unity/pubserv/csgo/addons/sourcemod/logs/errors_20200113.log")
L 01/13/2020 - 13:29:22: [SM] Exception reported: Failed to create native "BaseComm_IsClientGagged", name is probably already in use
L 01/13/2020 - 13:29:22: [SM] Blaming: basecomm.smx
L 01/13/2020 - 13:29:22: [SM] Call stack trace:
L 01/13/2020 - 13:29:22: [SM]   [0] CreateNative
L 01/13/2020 - 13:29:22: [SM]   [1] Line 71, /home/builds/sourcemod/linux-1.10/build/plugins/basecomm.sp::AskPluginLoad2
L 01/13/2020 - 13:29:22: [SM] Failed to load plugin "basecomm.smx": unexpected error 23 in AskPluginLoad callback.
L 01/13/2020 - 13:29:22: [SM] Exception reported: Failed to create native "GetAdminTopMenu", name is probably already in use
L 01/13/2020 - 13:29:22: [SM] Blaming: ma_adminmenu.smx
L 01/13/2020 - 13:29:22: [SM] Call stack trace:
L 01/13/2020 - 13:29:22: [SM]   [0] CreateNative
L 01/13/2020 - 13:29:22: [SM]   [1] Line 69, ma_adminmenu.sp::AskPluginLoad2
L 01/13/2020 - 13:29:22: [SM] Failed to load plugin "ma_adminmenu.smx": unexpected error 23 in AskPluginLoad callback.

```

**Comment by megoRU** [comment]
L 01/13/2020 - 13:29:22: [SM]   [1] Line 69, ma_adminmenu.sp::AskPluginLoad2
L 01/13/2020 - 13:29:22: [SM] Failed to load plugin "basecomm.smx": unexpected error 23 in AskPluginLoad callback.

**у вас 2 плагина одинаковых**

**Comment by valeologist** [comment]
> 
> 
> список плагинов:
> 
> ```
> 01 "Client Preferences" (1.10.0.6460) by AlliedModders LLC
> 02 "[Levels Ranks] Core" (v3.1.1) by RoadSide Romeo & Wend4r
> 03 "Knife ghost error fix" (1.1) by Phoenix & Wend4r
> 04 "Action if say.." (1.0.0) by OkyHp
> 05 "[Scoreboard] Language" (1.3) by Wend4r
> 06 "Reweapon" (1.1.0) by Phoenix, Tonki_Ton
> 07 "Anti-Flood" (1.10.0.6460) by AlliedModders LLC
> 08 "No Blood" (1.0.1) by Bara
> 09 "Ограничение оружия" (1.2.0) by Phoenix (˙·٠●Феникс●٠·˙)
> 10 "Basic Comm Control" (0.7.9) by AlliedModders LLC
> 11 "gravity.smx"
> 12 "Sound Commands" (1.10.0.6460) by AlliedModders LLC
> 13 "Реклама" (1.4) by Pheonix (˙·٠●Феникс●٠·˙)
> 14 "Basic Chat" (1.10.0.6460) by AlliedModders LLC
> 15 "Material Admin Checker" (0.7.9) by psychonic, Ca$h Munny, Sarabveer(VEER™)
> 16 "Weapon Skin" (2.3.10 PRIVATE) by Phoenix (˙·٠●Феникс●٠·˙)
> 17 "Map configs" (1.3) by Berni
> 18 "[LR] Module - Calibration" (v3.1) by RoadSide Romeo
> 19 "[LR WEB] Chat" (1.0.0) by OkyHp, SAPSAN
> 20 "[LR] Module - Unusual Kills" (v3.1) by Wend4r
> 21 "[LR] Module - FakeRank" (v3.1) by RoadSide Romeo & Wend4r
> 22 "[LR] Module - ExStats Hits" (v3.1) by Wend4r
> 23 "[LR] Module - ExStats Maps" (v3.1) by RoadSide Romeo
> 24 "[LR] Module - Top Vip Score" (v3.1) by Designed (Discord: .Designed#7985)
> 25 "[LR] Module - Overlays" (v3.1) by RoadSide Romeo
> 26  "[LR] Module - ExStats GeoIP" (v3.1) by RoadSide Romeo
> 27 "[LR] Module - WEB" (v2.5.4.2) by Wend4r
> 28 "[LR] Module - ExStats Weapons" (v3.1) by RoadSide Romeo & Wend4r
> 29 "Gloves" (1.6.1) by Phoenix (˙·٠●Феникс●٠·˙)
> 30 "[CS:GO] ResetScore" (1.0) by pracc
> 31 "Quake Sounds v3" (3.5.0) by Spartan_C001
> 32 "Player Commands" (1.10.0.6460) by AlliedModders LLC
> 33 "Extended Player Death Info" (1.1.0) by Phoenix (˙·٠●Феникс●٠·˙), Icons created Bastaz
> 34 "Reserved Slots" (1.10.0.6460) by AlliedModders LLC
> 35 "Knife choice CORE - Выбор ножа ЯДРО" (2.11) by Phoenix - Феникс
> 36 "Special Team Balancer" (0.6.0) by AlmazON
> 37 "[Gifts] Core" (3.2) by R1KO
> 38 "GameVoting" (1.1) by Neatek
> 39 "Fun Commands" (1.10.0.6460) by AlliedModders LLC
> 40 "Admin File Reader" (1.10.0.6460) by AlliedModders LLC
> 41 "Fun Votes" (1.10.0.6460) by AlliedModders LLC
> 42 "Basic Commands" (1.10.0.6460) by AlliedModders LLC
> 43 "IP Name & Say Block" (2.0) by R1KO
> 44 "bot_quota_0.smx"
> 45 "Material Admin" (0.7.9) by Material Admin Dev Team
> 46 "Admin Help" (1.10.0.6460) by AlliedModders LLC
> 47 "Basic Votes" (1.10.0.6460) by AlliedModders LLC
> 48 "Basic Info Triggers" (1.10.0.6460) by AlliedModders LLC
> 49 "autodemo" (2.2.4) by mukunda
> 50 "res/precacher.smx"
> 51 "Steam Works Group Manager" (1.5) by Someone
> 52 "VIP Hitmarker" (1.0.6) by Samolet, OkyHp
> 53 "[VIP] Electro Effects" (1.0.3) by R1KO
> 54 "[VIP] Stats" (2.6) by R1KO 
> 55 "[VIP] GrenadeTrails" (1.0.1) by R1KO
> 56 "[VIP] Vips Online" (1.0.1) by R1KO (skype: vova.andrienko1)
> 57 "[VIP] Test" (1.0.3) by R1KO (skype: vova.andrienko1)
> 58 "[VIP] CHAT (+scp)" (3.1.1) by R1KO
> 59 "[VIP] Spawn Effects" (1.2) by Mr.ToNik
> 60 "[VIP] MKIT" (1.1.3) by Pheonix (edit Felya)
> 61 "[VIP] Skins" (1.0.5) by R1KO (skype: vova.andrienko1)
> 62 "[VIP] Shield" (1.3.3 [PUBLIC]) by Faya™ (DS: Faya™#8514)
> 63 "[VIP] Admin" (1.1) by R1KO
> 64 "[VIP] Drop Weapons" (1.1) by R1KO
> 65 "[VIP] Core" (3.0 R) by R1KO
> 66 "Nextmap" (1.10.0.6460) by AlliedModders LLC
> 67 "SteamWorks Additive Glider" (1.0) by Kyle Sanderson
> 68 "Knife choice INTERFASE MONUL - Выбор ножа МОДУЛЬ ИНТЕРФЕЙСА" (1.1) by Phoenix - Феникс
> 69 "Admin Menu" (0.7.9) by AlliedModders LLC
> 70 "[Discord] Admin Log" (0.3.1) by Lord FEAR
> 71 "[Discord] Admin Session Log" (1.0) by CrazyHackGUT aka Kruzya
> 72 "[Discord] Core" (1.0.8) by CrazyHackGUT aka Kruzya
> 73 "[Discord] SourceBans + SourceComms" (1.2.2) by CrazyHackGUT aka Kruzya
> ```
> 
> Версия sourcemod:
> `sourcemod-1.10.0-git6460-linux.tar`
> 
> Лог ошибок
> 
> ```
> L 01/13/2020 - 13:29:22: SourceMod error session started
> L 01/13/2020 - 13:29:22: Info (map "workshop/1177199224/de_dust2_2x2") (file "/home/unity/pubserv/csgo/addons/sourcemod/logs/errors_20200113.log")
> L 01/13/2020 - 13:29:22: [SM] Exception reported: Failed to create native "BaseComm_IsClientGagged", name is probably already in use
> L 01/13/2020 - 13:29:22: [SM] Blaming: basecomm.smx
> L 01/13/2020 - 13:29:22: [SM] Call stack trace:
> L 01/13/2020 - 13:29:22: [SM]   [0] CreateNative
> L 01/13/2020 - 13:29:22: [SM]   [1] Line 71, /home/builds/sourcemod/linux-1.10/build/plugins/basecomm.sp::AskPluginLoad2
> L 01/13/2020 - 13:29:22: [SM] Failed to load plugin "basecomm.smx": unexpected error 23 in AskPluginLoad callback.
> L 01/13/2020 - 13:29:22: [SM] Exception reported: Failed to create native "GetAdminTopMenu", name is probably already in use
> L 01/13/2020 - 13:29:22: [SM] Blaming: ma_adminmenu.smx
> L 01/13/2020 - 13:29:22: [SM] Call stack trace:
> L 01/13/2020 - 13:29:22: [SM]   [0] CreateNative
> L 01/13/2020 - 13:29:22: [SM]   [1] Line 69, ma_adminmenu.sp::AskPluginLoad2
> L 01/13/2020 - 13:29:22: [SM] Failed to load plugin "ma_adminmenu.smx": unexpected error 23 in AskPluginLoad callback.
> ```
